src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -58,21 +60,30 @@ const ProgressiveProfiling = (props) => {
       })
       .catch(() => {});
 
-    let userEnrollmentAction = false;
     if (registrationResponse) {
       setRegistrationResult(registrationResponse);
       sendPageEvent('login_and_registration', 'welcome');
-
-      const queryParams = getAllPossibleQueryParams(registrationResponse.redirectUrl);
-      if ('enrollment_action' in queryParams) {
-        userEnrollmentAction = true;
-      }
     }
+  }, [DASHBOARD_URL, registrationResponse]);
 
-    if (enablePersonalizedRecommendations && !userEnrollmentAction) {
-      setShowRecommendationsPage(true);
+  useEffect(() => {
+    let queryParams = {};
+    let timer = null;
+    if (registrationResponse) {
```

Comment:
```suggestion
    if (registrationResponse) {
      let queryParams = {};
      let timer = null;
```

Reply:
we cant do that for timer, we need it outside the if block end, therefore I kept both timer and queryParams together.  should I move queryParams inside if?

Reply:
I guess than outside is fine